incremental-font-transfer/src/patch_group.rs

```diff
@@ -153,7 +153,8 @@ impl PatchGroup<'_> {
             partial_invalidation_iftx,
             mut no_invalidation_ift,
             mut no_invalidation_iftx,
-        } = GroupingByInvalidation::group_patches(candidates, ift_compat_id, iftx_compat_id);
+        } = GroupingByInvalidation::group_patches(candidates, ift_compat_id, iftx_compat_id)
+            .map_err(|_| ReadError::MalformedData("Malformed URI templates."))?;
```

Comment:
we have an inner error here that should provide more detail, why not preserve that in a `BadUri` error variant?

Reply:
I'm not sure I want to add a ReadError type that's specific to IFT since ReadError is more general than IFT. MalformedData does capture the essence of the error and in this case the inner type doesn't have any additional information that the error string doesn't already capture.